docs/dev/tags.rst

```diff
@@ -0,0 +1,27 @@
+
+
+Basic process of adding and using a new tag:
+--------------------------------------------
+
+To create a *global tag* (one applies to all programs), add a new line to the global tag dictionary ``all_global_tags`` in the
+file https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/tagdict/__init__.py.
+To create a *program-specific tag*, add the new tag's name to the program tag dictionary ``all_program_tags`` in the same file as above.
+Program tags are used with ``Tag.getProgramTag()`` or ``Tag.getBooleanTag()`` with a program argument in place of a boolean one in the
+first coordinate.
```

Comment:
I don't think the last part of this parses -- I'm not quite sure what you mean actually.